File: python/faktor.py
#!/usr/bin/env python3
""" Solve the Faktor problem on kattis """

import sys

# Given rounding up, just deduct 1 from the target case,
# Multiply the first by the modified second, and then add 1 to the result

data = sys.stdin.read().strip().split(" ")
data[0] = float(data[0])
data[1] = float(data[1]) - 1
result = data[0] * data[1]
print(int(result + 1))

File: c/faktor.c
/* A solution to the Faktor problem on Kattis */
#include <stdio.h>
int main(void){
    float articles, impact;
    while(scanf("%f %f", &articles, &impact) == 2){
        // Because we're rounding down, we can just decrement the target, multiply the first by it and add the 1 back in again
        impact--;
        float result = (articles * impact) + 1;
        printf("%d", (int)result);
    }
}
